massa-execution/src/sce_ledger.rs

```diff
@@ -413,12 +412,21 @@ impl SCELedger {
     }
 }
 
+/// The final ledger.
+#[derive(Debug, Clone)]
+pub struct FinalLedger {
+    /// The slot of the ledger.
```

Comment:
The state of the ledger is just after the execution if that slot right ?

Reply:
done